### PR TITLE
Feature/app redesign/gen 689 insurance/gen 705 update card

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## [APP-XXX]

- Updated card to reflect new design

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Contracts/Sources/View/ContractRow.swift [hunk 1]
```diff
@@ -105,38 +87,23 @@ struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
...
-                            )
-                    } else {
-                        DetailPill(text: pill)
-                    }
-                }
-                Spacer()
-                ContractRowChevron()
-            }
+            hText(contract.getDetails())
+                .foregroundColor(hTextColorNew.negative.opacity(0.415))
```
Comment (julia-andersson) [inline Projects/Contracts/Sources/View/ContractRow.swift:94]: Should be hGrayscaleTranslucentDarkColorNew.greyScaleTranslucent700

File: Projects/Contracts/Sources/View/ContractRow.swift [hunk 2]
```diff
@@ -105,38 +87,23 @@ struct ContractRowButtonStyle: SwiftUI.ButtonStyle {
...
-                                    .stroke(hLabelColor.primary, lineWidth: 1)
-                            )
-                    } else {
-                        DetailPill(text: pill)
-                    }
-                }
-                Spacer()
-                ContractRowChevron()
-            }
+            hText(contract.getDetails())
```
[the remark this answers is not in the document]
Comment (sladan-hedvig) [reply Projects/Contracts/Sources/View/ContractRow.swift:93]: Yes, but we are getting it like that from BE. I think they will fix this. Need to have a meeting after we finish everything with redesign of insurance.

File: Projects/Contracts/Sources/View/ContractRow.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/Contracts/Sources/View/ContractRow.swift]: Updated color :D

File: Projects/Contracts/Sources/View/ContractRow.swift (hunk 2 above)
[the remark this answers is not in the document]
Comment (julia-andersson) [reply Projects/Contracts/Sources/View/ContractRow.swift:93]: Oki!

**Review by julia-andersson — APPROVED** [review]

**Pull request merged**